### PR TITLE
Add cy.session to Cypress auth

### PR DESCRIPTION
**Pull request by matejnesuta**
Related to #7018 

Working on every platform mentioned in the issue except for ibmcloud, which I don't have access to.

### CODE REVIEW

File: frontend/cypress/support/commands.ts [hunk 1]
```diff
@@ -114,9 +100,8 @@ const finishLogin = (authEndpoint: string, username: string, password: string, c
       csrf: csrf
     }
   }).then(resp => {
-    const kialiURLWithToken = new URL(resp.redirects[1].replace('302: ', ''));
```
Comment (matejnesuta) [inline frontend/cypress/support/commands.ts:117]: I replaced this with .pop(), as the previous version threw some errors on ROSA and this seemed to fix things.

**Comment by ScriptingShrimp** [comment]
I will test this on crc, kind and on remote OCP. We also should cherry pick this into v1.73 right after PR https://github.com/kiali/kiali/pull/7104

**Comment by ScriptingShrimp** [comment]
passing on OCP, PR https://github.com/kiali/kiali/pull/7104 merged. 
@matejnesuta were you able to test this on CRC (Openshift local)?

**Review by nrfox — APPROVED** [review]
LGTM thanks for picking this up

**Review by ScriptingShrimp — APPROVED** [review]
LGTM

**Pull request merged**